adapters/socketio_berkeley.c

```diff
@@ -485,14 +483,10 @@ static NETWORK_INTERFACE_DESCRIPTION* create_network_interface_description(struc
         destroy_network_interface_descriptions(result);
         result = NULL;
     }
-    else if (strcpy(result->name, ifr->ifr_name) == NULL)
-    {
-        LogError("failed setting interface description name (strcpy failed)");
```

Comment:
Is there any way we should still be checking for these error states (like result->name != NULL after the strcpy), or would that not be checking anything either?

Reply:
That would not be checking anything. 
According to the C99 spec all `strcpy` does it to return the pointer of the first argument. There are no failures reported by this function.

Reply:
result->name is the same before and after the call. The only protections is to use strncpy just incase the math on line 482 is wrong.